### PR TITLE
Misc. typos (cont.)

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -I ../pcl-whitelist.txt`

Whitelist:
```
ang
childs
indeces
ith
lod
metre
metres
nd
ot
te
vertexes
```

### CODE REVIEW

**Comment by luzpaz** [comment]
Commits in 009cb02c9a40a41237d7a5a8df403ce645640b7b need a little more extra review since source changes have been made.

**Comment by taketwo** [comment]
👍 

There is one function change, but it is an util function in GPU module. I don't think we need to go through the deprecation procedure for that.

File: gpu/features/src/spinimages.cu [hunk 1]
```diff
@@ -244,7 +244,7 @@ namespace pcl
 					const float *amgles_end = amgles_beg + FSize;
 					const float *images_beg = simage_angles;
 
-					Block::transfrom(amgles_beg, amgles_end, images_beg, output.ptr(i_input), Div12eps());
+					Block::transform(amgles_beg, amgles_end, images_beg, output.ptr(i_input), Div12eps());
```
Comment (SergioRAgostinho) [inline gpu/features/src/spinimages.cu]: I just noticed the `amgles` use all around. Could you also address this one?

**Comment by luzpaz** [comment]
pushed requested revision. (FYI, there are more typos but too many to include in this PR. Stay tuned for follow-ups)

**Comment by taketwo** [comment]
I didn't know we have a guideline on how many typos can be fixed in a single PR :😂

**Comment by luzpaz** [comment]
> I didn't know we have a guideline on how many typos can be fixed in a single PR 😂

```
[foo@bar pcl]$ codespell -q 3 -I ../pcl-whitelist.txt --skip="./surface/include/pcl/surface/3rdparty" | wc -l
461
```
I didn't want to overwhelm. :)

**Pull request merged**